### PR TITLE
fix: Filtering is still able to be done when only one state is available

### LINKED ISSUE #7018: Filtering is still able to be done when only one state is available
This probably is a bit of a stress case to work to so might be a low priority. You can currently filter by one team meet when you only have one, which feels 'odd'.

2022-08-07 at 13 13

I would recommend when you only have a single state possible, filtering just doesn't show. This feels a more expected interface behaviour.

### PR DESCRIPTION
**Pull request by gitstart**
# Description
Filtering does not show if there is only one team member.
Filtering only shows if there is two or more team members.
Fixes/Partially Fixes #7018

# Loom video
https://www.loom.com/share/33e002ee39584a0fb70b02410fea8a84

## Testing scenarios

- [ ] Scenario A
  - Step 1
  - Start with one team member
  - Step 2
  - Start with two or more team members

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by gitstart** [comment]
Hi @Dschoordsch, @igorlesnenko your reviews are required for this PR

**Comment by Dschoordsch** [comment]
@JimmyLv can you please review?

**Comment by Dschoordsch** [comment]
Realized you were on a couple of other review requests already.
@igorlesnenko could you please review?

File: packages/client/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.tsx [hunk 1]
```diff
@@ -156,14 +156,18 @@ const TeamTasksHeader = (props: Props) => {
...
-        <DashFilterToggle
-          label='Team Member'
-          onClick={togglePortal}
-          onMouseEnter={TeamDashTeamMemberMenu.preload}
-          ref={originRef}
-          value={teamMemberFilterName}
-        />
-        {menuPortal(<TeamDashTeamMemberMenu menuProps={menuProps} team={team} />)}
+        <TeamMeta>
+          {teamMembers.length && (
```
Comment (igorlesnenko) [inline packages/client/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.tsx:160]: The condition should probably be `teamMembers.length` > 1 as I still able to see the filter with one member in the team

image

**Review by igorlesnenko — CHANGES_REQUESTED** [review]
Still able to see the filter

image

**Review by igorlesnenko — APPROVED** [review]

**Review by Dschoordsch — APPROVED** [review]

**Comment by Dschoordsch** [comment]
Did a quick test

**Pull request merged**